.cirrus.yml

```diff
@@ -372,6 +383,7 @@ task:
 
       build_script: su postgres -c 'ninja -C build -j${BUILD_JOBS}'
       build_32_script: su postgres -c 'ninja -C build-32 -j${BUILD_JOBS}'
+      install_script: su postgres -c 'ninja -C build install'
```

Comment:
Why do we need this install? At the very least we'd need different prefixes for the 32bit and the 64bit build.